src/components/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, ButtonGroup } from "@material-ui/core";
+import { useContext } from "react";
+import styled from "styled-components";
+
+import { ThemeContext } from "../contexts/ThemeContext";
+import { TranslationContext } from "../contexts/TranslationContext";
+import { themes } from "../types/ThemeTypes";
+
+const ThemeSwitcherStyled = styled(ButtonGroup)`
+  margin: 15px;
+  align-self: end;
+`;
+export default function ThemeSwitcher() {
+  const langConfig = useContext(TranslationContext).config;
```

Comment:
I don't really like this approach when wherever you'd like to use i18n, you'll have to drag a whole object instead of using path as a key without exposing the "config".

Reply:
dragging of all config also provides an interface for it, so it reduces the number of potential bugs. that's why I find this approach more comfortable for use.

Reply:
`dragging of all config also provides an interface for it` - could you please explain that part?
`it reduces the number of potential bugs` - what kinds of bugs are we talking about here?

Reply:
I'm talking about this interface: 
<img width="567" alt="Screenshot 2021-09-08 at 17 32 17" src="https://user-images.githubusercontent.com/33167093/132539583-cf9cc774-f508-48fc-97b2-4493f8b558fc.png">

this approach generates config interface directly from en.json file, therefore you don't need to create&support interfaces for paths, and it helps avoid all related bugs